src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java

```diff
@@ -108,7 +108,7 @@ public class TreeMojo extends AbstractMojo {
     /**
      * If specified, this parameter will cause the dependency tree to be written using the specified format. Currently
      * supported formats are: <code>text</code> (default), <code>dot</code>, <code>graphml</code>, <code>tgf</code>
-     * and <code>json</code>.
+     * and <code>json</code> (since 3.7.0).
```

Comment:
I don't know that we need this. It's implicitly the current version being described.